### PR TITLE
Update README

### LINKED ISSUE #17: Remove reference to gocsi
Reference to gocsi should be removed from README as gocsi is no longer maintained:

https://github.com/kubernetes-csi/csi-driver-image-populator#test-using-csc

### PR DESCRIPTION
**Pull request by f10atin9**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind documentation

**What this PR does / why we need it**:
Reference to gocsi should be removed from README as gocsi is no longer maintained.
**Which issue(s) this PR fixes**:

Fixes #17 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +65,6 @@ spec:
 $ sudo ./bin/imageplugin --endpoint tcp://127.0.0.1:10000 --nodeid CSINode -v=5
 ```
 
-### Test using csc
-Get ```csc``` tool from https://github.com/rexray/gocsi/tree/master/csc
-
 ### Mount the image
 $ csc -e tcp://127.0.0.1:10000 node publish abcdefg --vol-context image=kfox1111/misc:test --cap MULTI_NODE_MULTI_WRITER,block --target-path /tmp/csi
```
Comment (kfox1111) [inline README.md]: If csc tool is going away, then the subsequent lines describing csc should probably go too?

**Comment by kfox1111** [comment]
Looks good to me. Thanks. :)

/lgtm

**Comment by f10atin9** [comment]
> Looks good to me. Thanks. :)
> 
> /lgtm

My pleasure. : )

**Comment by f10atin9** [comment]
#17

**Pull request merged**